assets/stylesheets/bootstrap/_variables.scss

```diff
@@ -72,8 +72,8 @@ $font-size-small:         floor(($font-size-base * 0.8461538462)) !default; // ~
 $font-size-h1:            floor(($font-size-base * 2.3076923077)) !default; // ~30px
 $font-size-h2:            floor(($font-size-base * 2.3076923077)) !default; // ~30px
 
-$font-size-h3:            ceil(($font-size-base * 1.7)) !default; // ~20px
-$font-size-h4:            ceil(($font-size-base * 1.7)) !default; // ~20px
```

Comment:
Given that the comment suggested we got this wrong, is it some more general problem with the base font size, or just a mistake?